README.md

```diff
@@ -230,11 +230,12 @@ const utils: {
   mod: (number: number | bigint, modulo = CURVE.P): bigint;
   sha256: (message: Uint8Array) => Promise<Uint8Array>;
   hmacSha256: (key: Uint8Array, ...messages: Uint8Array[]) => Promise<Uint8Array>;
+  taggedHash: (tag: string, ...messages: Uint8Array[]) => Promise<Uint8Array>;
 
-  // You can set up your synchronous methods for `signSync` to work. The argument order is
-  // identical to async methods from above
-  sha256Sync: undefined;
+  // You can set up your synchronous methods for `signSync`/`signSchnorrSync` to work.
+  // The argument order is identical to async methods from above
   hmacSha256Sync: undefined;
+  taggedHashSync: undefined;
```

Comment:
I'll definitely change it if you want, but let me explain my thinking first:

I started out using the `sha256` sync/async, but it ended up rather ugly with having another layer of sync/async copy pasta.

My decision here may have been colored by the fact that I had a `taggedHash` synchronous implementation available: https://github.com/bitcoinjs/bitcoinjs-lib/blob/master/ts_src/crypto.ts#L55

I suspect that consumers of the bip340 schnorr signing are quite likely to have a `taggedHash` implementation because they're doing other tagged hashes when building Taproot trees and such.

We could encourage using `noble-secp256k1`'s `taggedHash` by exposing both sync and async versions, and switching back to accepting `string` from an enumerated type.

Reply:
I know, it may feel uncool, but let's keep the amount of stuff user needs to configure to a bare minimum. E.g. if we can say "all you need to do is re-defined sha256Sync" that's great.

Reply:
Do you want to keep `taggedHash`(/sync) taking the enumerated type, or switch to string and expose it?

Thanks!